API.md

```diff
@@ -63,115 +393,187 @@ This page documents the Party service API endpoints. All endpoints return an `HT
     "name": "Bolts and Ratchets Ltd"
 }
 ```
+<hr>
+
+#####Respondents Endpoints
+* `PUT /party-api/v1/respondents/email`
+* `PUT /party-api/v1/respondents/change_email`
+    * Changes a respondents old email address to a new provided one.
+        * Both the current email '`email_address`' and the '`new_email_address`' are to be provided in the body.
+    * Both URLs are linked to the same endpoint.
+    * This endpoint will update a respondent's email in the ras-party database and the oauth2 server. This email will need verified again so it will also set the user as unverified in the oauth server and will send a new verification email to the respondent.
+##### Example JSON DATA for the put
+```json
+{
+   "email_address": "old@email.com",
+   "new_email_address": "new@email.com"
+}
+```
 
-* `GET /businesses/id/d826818e-179e-467b-9936-6a8603dc8b46`
 
-&mdash; When a business is requested this returns a concrete representation of the business party. This representation will include any respondents associated with the business and any survey enrolments they have.
+##### Example JSON Response
+```json
+{
+   "emailAddress": "testtest@test.test",
+   "firstName": "Test",
+   "id": "ef7737df-2097-4a73-a530-e98dba7bfe43",
+   "lastName": "tseT",
+   "sampleUnitType": "BI",
+   "status": "ACTIVE",
+   "telephone": "07846608000"
+}
+```
+<hr>
 
-### Example JSON Response
+* `POST /party-api/v1/respondents/add-survey`
+    * This endpoint will enrol a respondent in a survey and associate with business if not already associated.
 
+##### Example JSON data for post
 ```json
 {
-    "associations": [
-        {
-        "enrolments": [
-            {
-                "enrolmentStatus": "ENABLED",
-                "surveyId": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
-            }
-        ],
-        "partyId": "cd592e0f-8d07-407b-b75d-e01fbdae8233",
-        "businessRespondentStatus": "ACTIVE"
-        }
-    ],
-    "name": "Bolts and Ratchets Ltd",
-    "id": "b3ba864b-7cbc-4f44-84fe-88dc018a1a4c",
-    "sampleUnitRef": "50012345678",
-    "sampleUnitType": "B"
+    "party_id": "438df969-7c9c-4cd4-a89b-ac88cf0bfdf3",
+    "enrolment_code": "fb747cq725lj"
+}
+``` 
+
+<hr>
+
+* `PUT /party-api/v1/respondents/edit-account-status/<respondent_party_id>`
+    * This endpoint will change a respondent's account status based on status_change variable.
```

Comment:
Isnt this duplicated on line 663?

Reply:
Should mention the valid new status values ( ACTIVE or SUSPEND I think)

Reply:
Adde and deleted duplicate